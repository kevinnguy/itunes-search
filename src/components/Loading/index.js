import React from 'react';
import { Image, StyleSheet } from 'react-native';

export default function Loading({ style }) {
  return (
    <Image
      style={[styles.container, style]}
      source={{
        uri:
          'https://www.pngitem.com/pimgs/m/235-2350144_load-map-bar-ios-5s-iphone-progress-clipart.png',
      }}
    />
  );
}

const styles = StyleSheet.create({
  container: {
    height: 40,
    width: 40,
  },
});
