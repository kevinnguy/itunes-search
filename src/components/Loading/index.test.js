import React from 'react';
import renderer from 'react-test-renderer';

import Loading from './index';

test('renders correctly', () => {
  const tree = renderer.create(<Loading />).toJSON();
  expect(tree).toMatchSnapshot();
});
