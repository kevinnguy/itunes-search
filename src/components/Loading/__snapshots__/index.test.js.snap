// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Image
  source={
    Object {
      "uri": "https://www.pngitem.com/pimgs/m/235-2350144_load-map-bar-ios-5s-iphone-progress-clipart.png",
    }
  }
  style={
    Array [
      Object {
        "height": 40,
        "width": 40,
      },
      undefined,
    ]
  }
/>
`;
